### PR TITLE
Psp 2331

### PR DESCRIPTION
**Pull request by devinleighsmith**
image
image

### CODE REVIEW

**Comment by asanchezr** [comment]
Looks like this branch was out of date? I see many changes that were already brought in in a previous PR...

File: frontend/src/features/leases/detail/LeasePages/deposits/Deposits.test.tsx [hunk 1]
```diff
@@ -69,6 +66,12 @@ const setup = (renderOptions: RenderOptions & { lease?: IFormLease } = {}): Rend
 };
 
 describe('Lease Deposits', () => {
+  beforeEach(() => {
```
Comment (devinleighsmith) [inline frontend/src/features/leases/detail/LeasePages/deposits/Deposits.test.tsx:69]: @asanchezr have to watch tests that take snapshots and use moment() or now.

**Review by FuriousLlama — APPROVED** [review]

File: frontend/src/features/leases/detail/LeasePages/deposits/Deposits.test.tsx (hunk 1 above)
Comment (asanchezr) [reply frontend/src/features/leases/detail/LeasePages/deposits/Deposits.test.tsx:69]: I see - good catch!

File: frontend/src/AppRouter.tsx [hunk 2]
```diff
@@ -133,11 +135,20 @@ const AppRouter: React.FC = () => {
         />
         <AppRoute
           protected
-          path="/lease/:leaseId?"
+          path="/lease/new"
+          exact
+          component={DetailContainer}
+          layout={AuthLayout}
+          claim={Claims.PROPERTY_VIEW}
```
Comment (asanchezr) [inline frontend/src/AppRouter.tsx:142]: Is this the right claim? Would expect it to be `Claims.LEASE_CREATE`....

File: frontend/src/AppRouter.tsx [hunk 3]
```diff
@@ -133,11 +135,20 @@ const AppRouter: React.FC = () => {
...
+          title={getTitle('Create/Edit Lease & Licenses')}
+        />
+        <AppRoute
+          protected
+          path="/lease/:leaseId"
           component={LeaseContainer}
           layout={AuthLayout}
           claim={Claims.PROPERTY_VIEW}
```
Comment (asanchezr) [inline frontend/src/AppRouter.tsx:150]: Same comment re: claim here

File: frontend/src/features/leases/index.ts [hunk 4]
```diff
@@ -1,3 +1,6 @@
+export * from './add/AddLeaseLayout';
+export * from './add/DetailContainer';
+export * from './add/DetailContainer';
```
Comment (asanchezr) [inline frontend/src/features/leases/index.ts:3]: Duplicated line here: copy/paste?

File: frontend/src/AppRouter.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/AppRouter.tsx:142]: True, will correct.

File: frontend/src/features/leases/index.ts (hunk 4 above)
Comment (devinleighsmith) [reply frontend/src/features/leases/index.ts:3]: I'm having some issues with my vscode dependency plugin, will fix regardless.

**Review by asanchezr — APPROVED** [review]

**Pull request merged**